### PR TITLE
YAL ContentRepo Banner Messages

### PR DESCRIPTION
**Pull request by rudigiesler**
If there are any messages tagged with "banner" in the content repo, we want to send that content as additional messages after the main menu

### CODE REVIEW

File: vaccine/tests/test_worker.py [hunk 1]
```diff
@@ -407,7 +407,7 @@ async def test_answer_worker_push_results_user_error(
     )
 
     # wait for worker to log error
-    for _ in range(10):
+    for _ in range(50):
```
Comment (rudigiesler) [inline vaccine/tests/test_worker.py:410]: Make this test more reliable

File: yal/contentrepo.py [hunk 2]
```diff
@@ -171,7 +181,7 @@ async def get_page_details(user, page_id, message_id, suggested=False):
             except HTTP_EXCEPTIONS as e:
                 if i == 2:
                     logger.exception(e)
-                    return True, []
+                    return True, {}
```
Comment (rudigiesler) [inline yal/contentrepo.py:184]: There wasn't any typing on these functions, so I added some, which revealed some issues that I also fixed

File: yal/tests/conftest.py [hunk 3]
```diff
@@ -0,0 +1,9 @@
+from unittest import mock
+
+import pytest
+
+
+@pytest.fixture(autouse=True, scope="package")
+def mock_sleep():
+    with mock.patch("asyncio.sleep"):
+        yield
```
Comment (rudigiesler) [inline yal/tests/conftest.py:9]: This is to speed up tests, so that we don't need to wait for all the `sleep` calls

**Review by KaitCrawford — APPROVED** [review]
👍 Looks great!

**Pull request merged**